Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -75,284 +76,169 @@ public class ProjectController : Controller
-                ex.ToExceptionless().Submit();
-                throw ex;
+                else
+                { return Ok(project); }
             }
+            return Ok(false);
         }
 
         /**
-        * @api {put} api/Project/editProject 
+        * @api {put} api/project 
         * @apiVersion 1.0.0
-        * @apiName Project
+        * @apiName EditProject
         * @apiGroup Project
-        *  @apiParam {int} Id  Project Id
-        * @apiParam {string} Name  Project Name
-        * @apiParam {string} SlackChannelName  Project SlackChannelName
-        * @apiParam {bool} IsActive  Project IsActive
-        * @apiParam {int} TeamLeaderId  Project TeamLeaderId
-        * @apiParam {UserAc} ApplicationUsers  Project ApplicationUsers
+        * @apiParam {object} ProjectAc object 
         * @apiParamExample {json} Request-Example:
-        *      
-        *        {
-        *             "Id":"1",
-        *             "Name":"ProjectName",
-        *             "SlackChannelName":"SlackChannelName",
-        *             "IsActive":"True",
-        *             "TeamLeaderId":"1",
-        *             "ApplicationUsers":"List of Users"
-        *        }      
+        * {
+        *   "Id":"1",
+        *   "Name":"ProjectName",
+        *   "SlackChannelName":"SlackChannelName",
+        *   "IsActive":"True",
+        *   "TeamLeaderId":"1",
+        *   "ApplicationUsers":"List of Users"
+        * }      
         * @apiSuccessExample {json} Success-Response:
         * HTTP/1.1 200 OK 
         * {
-        *     "description":"edit Project in ProjectTable"
+        *     "description":"Update project information"
         * }
         */
         [Authorize]
         [HttpPut]
-        [Route("editProject")]
-        public async Task<IActionResult> editProject(int id, [FromBody]ProjectAc project)
+        [Route("")]
+        public async Task<IActionResult> EditProject([FromBody]ProjectAc project)
         {
-            try
+            var updatedBy = _userManager.GetUserId(User);
+            if (ModelState.IsValid)
             {
-                var updatedBy = _userManager.GetUserId(User);
-
-                if (ModelState.IsValid)
+                ProjectAc projectAc = await _projectRepository.CheckDuplicateProject(project);
+                if (!string.IsNullOrEmpty(projectAc.Name) && !string.IsNullOrEmpty(projectAc.SlackChannelName))
                 {
-                    ProjectAc projectAc = _projectRepository.checkDuplicateFromEditProject(project);
-                    if (projectAc.Name != null && projectAc.SlackChannelName != null)
-                    {
-                        await _projectRepository.EditProject(project, updatedBy);
-                    }
-                    else { return Ok(project); }
+                    await _projectRepository.EditProject(project, updatedBy);
                 }
-                return Ok(project);
+                else { return Ok(project); }
```

Comment:
if Project does not exists or bad information is sent. It should return either NotFound or BadRequest.